src/main/java/com/easypost/model/Shipment.java

```diff
@@ -1062,4 +1062,26 @@ public Rate lowestRate(final List<String> carriers, final List<String> services)
 
         return lowestRate;
     }
+
+    /**
+     * Get the rates of a shipment.
+     *
+     * @return Rate object
+     * @throws EasyPostException
+     */
+    public Shipment retrieveRates() throws EasyPostException {
+        return retrieveRates(null);
+    }
+
+    /**
+     * Get the rates of a shipment.
+     *
+     * @return Rate object
+     * @param apiKey API key to use in request (overrides default API key).
+     * @throws EasyPostException
+     */
+    public Shipment retrieveRates(String apiKey) throws EasyPostException {
```

Comment:
Thinking on the usage of this function tonight. Will get back to you tomorrow (static vs non-static - it may need to be adjusted across all our libraries).